apps/crn-server/test/routes/manuscript.route.test.ts

```diff
@@ -348,6 +454,95 @@ describe('/manuscripts/ route', () => {
       expect(response.status).toEqual(403);
     });
 
+    test('Should return a Contentful error when Contentful API fails', async () => {
+      const createManuscriptRequest: ManuscriptPostRequest =
+        getManuscriptPostBody();
+
+      userMockFactory.mockReturnValueOnce(createUserResponse());
+
+      const contentfulError = {
+        status: 400,
+        message: 'Invalid request to Contentful',
+        details: {
+          errors: [{ message: 'Field X is required' }],
+        },
+      };
+
+      manuscriptControllerMock.createVersion.mockRejectedValueOnce(
+        new Error(JSON.stringify(contentfulError)),
+      );
+
+      const response = await supertest(app)
+        .post(`/manuscripts/${manuscriptId}`)
+        .send(createManuscriptRequest)
+        .set('Accept', 'application/json');
+
+      expect(response.status).toBe(400);
+      expect(response.body).toEqual({
+        message: 'Invalid request to Contentful',
+        errors: contentfulError.details.errors,
+      });
+    });
+
+    test('Should throw a generic error when an unexpected error occurs', async () => {
+      const createManuscriptRequest: ManuscriptPostRequest =
+        getManuscriptPostBody();
+
+      userMockFactory.mockReturnValueOnce(createUserResponse());
+
+      manuscriptControllerMock.createVersion.mockRejectedValue(
+        new Error('Unexpected error'),
+      );
+
+      const response = await supertest(app)
+        .post(`/manuscripts/${manuscriptId}`)
+        .send(createManuscriptRequest)
+        .set('Accept', 'application/json');
+
+      expect(response.status).toBe(500);
+
+      expect(response.body).toHaveProperty('message', 'Unexpected error');
+    });
+
+    test('Should throw a generic error when an unexpected error occurs', async () => {
+      const createManuscriptRequest: ManuscriptPostRequest =
+        getManuscriptPostBody();
+
+      userMockFactory.mockReturnValueOnce(createUserResponse());
+
+      manuscriptControllerMock.createVersion.mockRejectedValue(
+        new Error('Unexpected error'),
+      );
+
+      const response = await supertest(app)
+        .post(`/manuscripts/${manuscriptId}`)
+        .send(createManuscriptRequest)
+        .set('Accept', 'application/json');
+
+      expect(response.status).toBe(500);
+
+      expect(response.body).toHaveProperty('message', 'Unexpected error');
+    });
+
```

Comment:
```suggestion
```

we have the same test for `post(`/manuscripts/${manuscriptId}` on lines 487:505 above 😉